packages/audioplayers/lib/src/log_level.dart

```
@@ -11,4 +11,15 @@ extension LogLevelExtension on LogLevel {
         return 0;
     }
   }
+
+  static LogLevel fromInt(int level) {
+    switch (level) {
+      case 2:
+        return LogLevel.info;
+      case 1:
+        return LogLevel.error;
+      default:
```

should it just error for invalid levels?
I normally prefer that to failing silently, unless there is a reason to do it